atlp-rwanda/e-commerce-ones-and-zeroes-fn
# [Finishes #187355052] Users should be presented with a reset password

### What does this PR do?
This PR, allows a user or seller to be able to reset their password

### How should it be tested

1. Clone the repo, navigate to the branch `git checkout  fx-reset-password`
2. After, install all dependencies by `npm install`
3. check `.env.example` and set the required variables
4. start the applicatiin by: `npm start`
5. after that, navigate to `/login`
6. continue the process, click on `forgot password? `

### Required screenshot

<img width="1440" alt="Screenshot 2024-06-22 at 12 34 19" src="https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/88989844/78f6ddde-ab17-44dc-bc2a-0bcfca1332cb">
<img width="1416" alt="Screenshot 2024-06-22 at 12 34 41" src="https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/88989844/acbde6f2-101b-4140-9447-4cac5428df40">
<img width="922" alt="Screenshot 2024-06-22 at 12 34 56" src="https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/88989844/04484d5a-7591-4f04-abbd-4bc389a254a0">
<img width="1405" alt="Screenshot 2024-06-22 at 12 36 18" src="https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/88989844/12baede6-b9d3-44ff-9fc1-342125d18596">
<img width="1435" alt="Screenshot 2024-06-22 at 12 36 31" src="https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/88989844/1ba6fda9-c3e3-4445-a643-4da2b95e79b7">
<img width="1178" alt="Screenshot 2024-06-22 at 12 46 38" src="https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-fn/assets/88989844/81ab5361-7c4a-41cf-b19b-c7773264a466">
